leakcanary-android-core/src/main/java/leakcanary/internal/DisplayLeakAdapter.kt

```diff
@@ -177,9 +178,10 @@ internal class DisplayLeakAdapter constructor(
 
     htmlString += "$INDENTATION${extra("Leaking: ")}$reachabilityString<br>"
 
-    if (retainedHeapByteSize != null) {
-      htmlString += "${INDENTATION}${extra("Retaining ")}$retainedHeapByteSize${extra(
-          " bytes in "
+    retainedHeapByteSize?.let {
+      val humanReadableRetainedHeapSize = humanReadableByteCount(it.toLong(), si = true)
+      htmlString += "${INDENTATION}${extra("Retaining ")}$humanReadableRetainedHeapSize${extra(
+          " in "
```

Comment:
I don't fully understand the exact goal of `extra()` and the impact of extracting size unit out of it.

Reply:
yeaaaah that's... a very unfortunate piece of code :) . That thing adds font tags. You did the right thing.